scripts/update-snippet-repository

```diff
@@ -2,19 +2,22 @@
 
 SNIPPET_REPO=/var/lib/jenkins/snippets
 SNIPPET_ROOT=system/configs
-SNIPPET_EXPORT_FILENAME=snippets
 LOCK_FILE=/run/elektra-update-snippet-repository.lock
+KEY_SLASH_REPLACED=$(echo "$ELEKTRA_REST_KEY" | tr '/' '_')
 
 export_snippets() {
-  # snippets.dump does not need to be exported, it is the source file
-  kdb export $SNIPPET_ROOT ni > $SNIPPET_EXPORT_FILENAME.ni.ini
```

Comment:
The idea was to have both: everything in a file and every file separated. Is there a specific reason why you removed the everything in a file?